src/dsp_tools/commands/excel2json/utils.py

```diff
@@ -39,6 +44,39 @@ def read_and_clean_excel_file(excelfile: str, sheetname: str | int = 0) -> pd.Da
     return read_df
 
 
+def read_and_clean_all_sheets_excelfile(excelfile: str) -> dict[str, pd.DataFrame]:
+    """
+    This function reads an Excel file with all its sheets,
+    if there is a ValueError then it patches the openpyxl part that creates the
+    error and opens it with that patch.
+    It cleans and then returns a dictionary with the dataframes.
+
+    Args:
+        excelfile: The name of the Excel file
+
+    Returns:
+        A dictionary with all the sheets as dataframes
```

Comment:
Could you tell that more precise? e.g. 

```suggestion
        All sheets of the excel file, in the form of a dictionary {sheet_name: dataframe}
```